src/HotChocolate/Pagination/src/Pagination.EntityFramework/CursorKey.cs

```diff
@@ -19,7 +20,23 @@ public object Parse(ReadOnlySpan<byte> cursorValue)
 
     public bool TryFormat(object entity, Span<byte> buffer, out int written)
     {
-        var key = Property.GetValue(entity)!;
+        var key = GetValue(entity);
+
+        if (key is null)
+        {
+            written = 0;
+            return true;
+        }
```

Comment:
This is a problem. The keys can theoretically be nullable. This of course does not work. The question is, how do we want to encode "null" ?

Reply:
if we introduce escaping for null we would need to ensure that every other occurrence is encoded ... we could introduce `\\` and `\null` :)